pyproject.toml

```diff
@@ -44,16 +47,24 @@ Documentation = "https://docs.dwavequantum.com"
 "Source Code" = "https://github.com/dwavesystems/dwave-optimization"
 
 [dependency-groups]
+blas = [
+    # Optional build requirement
+    "scipy_openblas64==0.3.30.0.8",
+]
+build = [
+    # Same as build-system.requires
+    "cython~=3.2.0",
+    "meson~=1.9.0",
+    "meson-python~=0.18.0",
+    "ninja~=1.13.0",
+]
```

Comment:
TODO: Add a `blas-build` option as well that just combined `build` and `blas`. Slightly simpler.